templates/index.html

```diff
@@ -92,7 +92,6 @@ <h5 class="modal-title">April 26, 2021 Updates</h5>
             <div class="navbar-nav">
                 <a class="nav-item nav-link" href="https://github.com/pioneers/runtime/wiki/Student-API" target="_blank">Robot API</a>
                 <a class="nav-item nav-link" href="https://github.com/pioneers/pimulator-js/blob/master/releasenotes.md" target="_blank">Release Notes</a>
-                <a class="nav-item nav-link" href="https://forms.gle/i4q48q7fxBxRxDSb6" target="_blank">Bug Reports/Feature Requests</a>
```

Comment:
Make sure to only commit changes that are relevant to your code. It seems that your branch is behind. Make sure to pull before committing.